lib/esp/resources/custom_signature/result.rb

```diff
@@ -9,89 +9,85 @@ class Result < ESP::Resource
-      #
-      # +attributes+ | Required | A hash of custom signature result attributes
-      #
-      # ===== Valid Attributes
-      #
-      # See {API documentation}[http://api-docs.evident.io?ruby#custom-signature-result-create] for valid arguments
-      #
-      # ==== Example
-      #
-      #  signature = "# Demo Ruby Signature\r\nconfigure do |c|\r\n  # Set regions to run in. Remove this line to run in all regions.\r\n  c.valid_regions     = [:us_east_1]\r\n  # Override region to display as global. Useful when checking resources\r\n  # like IAM that do not have a specific region.\r\n  c.display_as        = :global\r\n  # deep_inspection works with set_data to automically collect\r\n  # data fields for each alert. Not required.\r\n  c.deep_inspection   = [:users]\r\nend\r\n\r\n# Required perform method\r\ndef perform(aws)\r\n  list_users = aws.iam.list_users\r\n  count = list_users[:users].count\r\n\r\n  # Set data for deep_inspection to use\r\n  set_data(list_users)\r\n\r\n  if count == 0\r\n    fail(user_count: count, condition: 'count == 0')\r\n  else\r\n    pass(user_count: count, condition: 'count >= 1')\r\n  end\r\nend\r\n"
-      #  result = ESP::CustomSignature::Result.create(signature: signature, language: "ruby", region_id: 1, external_account_id: 1)
+      #   @return [ActiveResource::PaginatedCollection<ESP::CustomSignature::Result>]
 
-      # :method: save
-      # Create or update a CustomSignature::Result
-      #
-      # ===== Valid Attributes
+      # @!method self.create(attributes = {})
+      #   Create a CustomSignature::Result
+      #   *call-seq* -> +super.create(attributes={})+
+      #
+      #   @param attributes [Hash] Required hash of custom signature result attributes.
+      #     ===== Valid Attributes
+      #
+      #     See {API documentation}[http://api-docs.evident.io?ruby#custom-signature-result-create] for valid arguments
+      #   @return [ESP::CustomSignature::Result]
+      #   @example
+      #    signature = "# Demo Ruby Signature\r\nconfigure do |c|\r\n  # Set regions to run in. Remove this line to run in all regions.\r\n  c.valid_regions     = [:us_east_1]\r\n  # Override region to display as global. Useful when checking resources\r\n  # like IAM that do not have a specific region.\r\n  c.display_as        = :global\r\n  # deep_inspection works with set_data to automically collect\r\n  # data fields for each alert. Not required.\r\n  c.deep_inspection   = [:users]\r\nend\r\n\r\n# Required perform method\r\ndef perform(aws)\r\n  list_users = aws.iam.list_users\r\n  count = list_users[:users].count\r\n\r\n  # Set data for deep_inspection to use\r\n  set_data(list_users)\r\n\r\n  if count == 0\r\n    fail(user_count: count, condition: 'count == 0')\r\n  else\r\n    pass(user_count: count, condition: 'count >= 1')\r\n  end\r\nend\r\n"
+      #    result = ESP::CustomSignature::Result.create(signature: signature, language: "ruby", region_id: 1, external_account_id: 1)
+
+      # @method save
```

Comment:
Looks like yard is still parsing this as expected, but `@metho` is not listed as a valid tag.  Should be `@!method`.